src/test/kotlin/com/dwolla/util/JsonBodyTest.kt

```diff
@@ -0,0 +1,18 @@
+package com.dwolla.util
```

Comment:
Should we move this test to a new `/http` folder in the `test/kotlin/com/dwolla` directory to mimic the `main/kotlin/com/dwolla/http` directory path?

Reply:
Should be moved now!